Tamagotchi.Core/Configuration/DragonAgeingOptions.cs

```diff
@@ -1,10 +1,20 @@
-namespace Tamagotchi.Core.Configuration
+﻿namespace Tamagotchi.Core.Configuration
 {
-    public class DragonAgeingOptions
```

Comment:
This could be further simplified with the following code or something similar:
public record DragonAgeingOptions(int ChildAfter, int TeenAfter, int AdultAfter, int DeadAfter);

Reply:
This won't be possible due to serialization. I am sure there is a way with some proper R&D.
![image](https://user-images.githubusercontent.com/71598973/114306116-af500a00-9ad2-11eb-8eaa-32649144b1e3.png)